cmd/eksctl/main.go

```diff
@@ -31,6 +35,17 @@ func addCommands(rootCmd *cobra.Command, flagGrouping *cmdutils.FlagGrouping) {
 }
 
 func main() {
+
+	// Log internally-logged kubernetes errors uniformly
+	logK8sError := func(err error) {
+		// we need to go one level deeper than k8sruntime.HandleError
+		_, file, line, _ := runtime.Caller(1)
+		idx := strings.Index(file, "/k8s.io/")
+		finalFileLine := file[idx+1:] + ":" + strconv.Itoa(line)
```

Comment:
We don't log file paths in our own logs, let's drop it from here. We can discuss logging file paths separately.

Reply:
I disagree. Without the line you won't easily know where the internal Kubernetes error was logged. They are often cryptic and lack context

Reply:
What if we added this as a debug line? Would that be ok?